modules/azure/api_management_api_schema/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "resource_group_name" {
+  type        = string
+  description = "Name of the resource group."
+}
+
+variable "api_management_name" {
+  type        = string
+  description = "The name of the API management service."
+}
+
+variable "schema" {
```

Comment:
This var should be required, if all null then this module does nothing and is useless